test.cmd

```diff
@@ -0,0 +1,2 @@
+@echo off
+powershell -ExecutionPolicy ByPass -NoProfile -command "& """%~dp0eng\common\Build.ps1""" -restore -build -test %*"
```

Comment:
Overall the PR looks really good now. My remaining concern is around the expectations of this script. I would expect it to restore and build the repository and then invoke all the tests (currently only GenerateScriptTests.csproj). But in-fact this only restore, builds and invokes the test project. I think that's OK for now (given that the repository is currently not buildable on Windows) but I wanted to call that out. @MichaelSimons any concerns with the current design?

Reply:
I am fine with that.